app/human-detector-app/contexts/userContext.ts

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+import User from '../classes/User';
+
+// eslint-disable-next-line import/prefer-default-export
+export const UserContext = createContext<User | null>(null);
```

Comment:
We might not need a `UserContext`cus we can ideally just fetch it from the `BackendService`, but this is fine for now

Reply:
To me, I feel like its better having a `UserContext` since the `BackendContext` really should be just getting stuff from endpoints.  It probably makes the code more readable